### PR TITLE
[WIP] Handle multiple user to set premissions

### PR DESCRIPTION
**Pull request by gperdomor**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes 
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #612

This PR allo to set multiple users for <code>writable</code> task

### CODE REVIEW

File: recipe/common.php [hunk 1]
```diff
@@ -305,8 +315,8 @@
                             $hasfacl = run("getfacl -p $dir | grep \"^user:$httpUser:.*w\" | wc -l")->toString();
                             // Set ACL for directory if it has not been set before
                             if (!$hasfacl) {
-                                run("setfacl -R -m u:\"$httpUser\":rwX -m u:`whoami`:rwX $dir");
-                                run("setfacl -dR -m u:\"$httpUser\":rwX -m u:`whoami`:rwX $dir");
+                                run("$sudo setfacl -R $permissions $dirs");
```
Comment (oanhnn) [inline recipe/common.php]: This block in else block (`empty($sudo) = true`).

File: recipe/common.php [hunk 2]
```diff
@@ -273,27 +274,36 @@
...
+    $users = env('writable_users');
+    $permissions = '';
+
     if (!empty($dirs)) {
         try {
             if (null === $httpUser) {
                 $httpUser = run("ps aux | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d\  -f1")->toString();
             }
 
+            $users[] = $httpUser;
```
Comment (oanhnn) [inline recipe/common.php]: if writable users included deployer user and http user ?

**Comment by oanhnn** [comment]
Please add unit test

File: recipe/common.php (hunk 2 above)
Comment (gperdomor) [inline recipe/common.php]: <pre><code>if(!in_array($httpUser, $users)){
    $users[] = $httpUser;
}</code></pre>

The same for <code>`whoami`</code>. But, i move forward with this or close this and wait for your solution as you said in #612?

**Comment by elfet** [comment]
No progress. Close.

**Pull request closed without merging**